ui/hooks/gasFeeInput/useMaxFeePerGasInput.js

```diff
@@ -34,14 +38,17 @@ const getMaxFeePerGasFromTransaction = (transaction) => {
  *  method to update the setMaxFeePerGas.
  */
 export function useMaxFeePerGasInput({
-  supportsEIP1559V2,
+  editGasMode,
   estimateToUse,
   gasEstimateType,
   gasFeeEstimates,
   gasLimit,
   gasPrice,
+  supportsEIP1559V2,
   transaction,
 }) {
+  const swapCustomMaxFeePerGas = useSelector(getCustomMaxFeePerGas);
```

Comment:
Not following why this value comes from customMaxFeePerGas?

Reply:
for swaps I found that values of gas are used from `getCustomMaxFeePerGas` and also if we update the gas value update is made to `getCustomMaxFeePerGas` not transaction